api/internal/managers/app/config/config.go

```diff
@@ -102,6 +106,24 @@ func (m *appConfigManager) ValidateConfigValues(configValues types.AppConfigValu
 		}
 	}
 
+	// Set appropriate message based on error types
+	if ve != nil && len(ve.Errors) > 0 {
+		// Check if ALL errors are about required fields
+		allRequired := true
+		for _, err := range ve.Errors {
+			if !strings.Contains(err.Message, "is required") {
```

Comment:
why error message contains `is required`? i see that this is removed: `ErrConfigItemRequired = errors.New("item is required")`. i would also suggest using errors.Is instead of checking against a hardcoded string.

Reply:
Yeah, i think it should be in the FE. Any reason it needs to be in the BE?

Reply:
the feedback i got was to keep all validation in the backend, and i'm not sure if still being able to present this message using the frontend only would count as me doing additional "frontend validation"